addons/web/static/src/views/fields/properties/properties_field.xml

```diff
@@ -1,64 +1,87 @@
-                            class="btn btn-light text-muted text-break m-0"
-                            t-on-click="onPropertyCreate">
-                            <i class="fa fa-plus"/>
-                            Add a Property
-                        </button>
+                        <t t-else="">
+                            <t t-set="domId" t-value="generateUniqueDomID()"/>
+                            <label
+                                t-attf-class="o_field_property_label o_form_label text-break {{state.movedPropertyName === propertyConfiguration.name ? 'o_property_field_highlight' : ''}} d-flex align-items-center"
+                                t-att-for="domId">
+                                <b
+                                    t-if="propertyConfiguration.string &amp;&amp; propertyConfiguration.string.length"
+                                    t-out="propertyConfiguration.string"
+                                    class="text-900"/>
+                                <i
+                                    t-else="" class="o_field_property_empty_label">
+                                    New Property
+                                </i>
+                                <i
+                                    t-if="state.canChangeDefinition &amp;&amp; !props.readonly"
+                                    class="o_field_property_open_popover fa fa-pencil ms-2"
+                                    t-on-click="(event) => this.onPropertyEdit(event, propertyConfiguration.name)"/>
+                            </label>
+                            <div class="o_property_field_value">
+                                <PropertyValue
+                                    id="domId"
+                                    canChangeDefinition="state.canChangeDefinition"
+                                    comodel="propertyConfiguration.comodel || ''"
+                                    context="props.context"
+                                    domain="propertyConfiguration.domain || '[]'"
+                                    readonly="props.readonly"
+                                    selection="propertyConfiguration.selection"
+                                    string="propertyConfiguration.string"
+                                    tags="propertyConfiguration.tags"
+                                    type="propertyConfiguration.type"
+                                    value="propertyConfiguration.value"
+                                    onChange.bind="(value) => this.onPropertyValueChange(propertyConfiguration.name, value)"
+                                    onTagsChange.bind="(newTags, newValue) => this.onTagsChange(propertyConfiguration.name, newTags, newValue)"
+                                />
+                            </div>
+                        </t>
                     </div>
                 </div>
             </t>
-
```

Comment:
this space shouldn't be there at the first place so I take the opportunity here to remove it

anyway I don't like inconsistency, so as we now use `and` instead of `&amp;&amp;` I'll change all occurence of `&amp;&amp;` in the property fields code in a second commit, and I'll remove that line in that commit as well